requirements/environment.yml

```diff
@@ -1,6 +1,5 @@
 channels:
-  - defaults
```

Comment:
Perhaps this file should be renamed to `environment_docs.yml` given that it's only used for our docs now?

Reply:
I was thinking that it could be worthy to use it in the future in a contributing guide (which checking I think is missing). Also, checking I think the envrionment file currently being used to build the docs is at the docs folder: https://github.com/spyder-ide/qtawesome/blob/master/docs/environment.yml

Should I also update the channel definition there to also use the conda-forge channel?

Reply:
> I was thinking that it could be worthy to use it in the future in a contributing guide (which checking I think is missing).

I thought the same too. Then please open an issue so you don't forget to add a small contributing guide in the future.

> Also, checking I think the envrionment file currently being used to build the docs is at the docs folder: https://github.com/spyder-ide/qtawesome/blob/master/docs/environment.yml

Hehe, my bad. I thought we were using the environment in requirements for our docs as well.

> Should I also update the channel definition there to also use the conda-forge channel?

Sure, that's a good idea.